build-tools/src/scripts/Dockerfile.alpine.bootstrap

```diff
@@ -0,0 +1,50 @@
+# This is the skeleton of the APKBUILD file one can use to stub out stubborn packages:
```

Comment:
There used to be just `pkg/alpine/Dockerfile`. Now we also have this file, `build-tools/src/scripts/Dockerfile.alpine.bootstrap`. How is this different than that? How is it used? Why is the other one not enough?

Reply:
More importantly, is this consistently used on every platform? Or is this a "risc-v special"?

Reply:
This is a "riscv special" in so much as we haven't had a case where Alpine would exist on a platform when EVE needed it. riscv64 is the first case where we are literally building the universe from scratch and that include bootstrapping "just enough Alpine itself" in parallel. This new Docker file is for THAT -- it is only applicable for bootstrapping Alpine. Still, I'd rather keep it in our repo as an example of how to do that once we need to support yet another brand new platform (like whatever quantum computing running Linux would be called ;-)).

Reply:
> whatever quantum computing running Linux would be called

qinux?

> This is a "riscv special" in so much as we haven't had a case where Alpine would exist on a platform when EVE needed it

So this isn't a riscv special, as much as a "when you already have base alpine for the target arch, use the `Dockerfile`; when you don't use the bootstrap."

Can we improve the attached docs to make that explicit?